recipes-devtools/deepstream/deepstream-5.0_5.0.0-1.bb

```diff
@@ -0,0 +1,73 @@
+DESCRIPTION = "NVIDIA Deepstream SDK"
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = "file://usr/share/doc/deepstream-5.0/copyright;md5=f635f9f375e764ce281a2070599e2457"
+
+SRC_URI="https://repo.download.nvidia.com/jetson/common/pool/main/d/deepstream-5.0/deepstream-5.0_5.0.0-1_arm64.deb;subdir=${BPN}"
+SRC_URI[sha256sum] = "a7a7015515883ac88c7587c7a2acfcf78510e539b84b702afd05f4f330faa55e"
+
+COMPATIBLE_MACHINE = "(tegra)"
+PACKAGE_ARCH = "${SOC_FAMILY_PKGARCH}"
+
+DEPENDS = "gstreamer1.0 gstreamer1.0-rtsp-server tensorrt cudnn libcublas cuda-cudart tegra-libraries"
+
+inherit pkgconfig
```

Comment:
The build for this recipe doesn't appear to use `pkg-config`, so this inherit isn't needed.